docs/development_guide.md

```diff
@@ -41,18 +41,20 @@ git remote add upstream git@github.com:googlegenomics/gcp-variant-transforms.git
 
 #### Setup virtualenv
 
+Ensure you are using Python 3.7 version, since Apache Beam does not support 3.8.
+
 ```bash
-sudo apt-get install python-pip python-dev build-essential
-sudo python -m pip install --upgrade pip
-sudo python -m pip install --upgrade virtualenv
+sudo apt-get install python3-pip python3-venv python3.7-venv python-dev build-essential
+sudo python3 -m pip install --upgrade pip
+sudo python3 -m pip install --upgrade virtualenv
```

Comment:
Let's make this almost identical to our dockerfile:
```
    sudo apt-get install python3-pip python3-venv python-dev build-essential
    python3 -m venv venv3
    . venv3/bin/activate
    python3 -m pip install --upgrade pip
    python3 -m pip install --upgrade wheel
    python3 -m pip install --upgrade .
```

Unless there is a reason we should not do this and we need to update python system-wide.